test/functorch/test_control_flow.py

```diff
@@ -248,6 +251,7 @@ def f(x, pred, pred2):
         graph = make_fx(f, tracing_mode="symbolic")(x, torch.tensor(False), torch.tensor(False))
         self.assertEqual(graph(x, torch.tensor(True), torch.tensor(True)), f(x, torch.tensor(True), torch.tensor(True)))
 
+    @unittest.expectedFailure
```

Comment:
What's the problem with cond and functionalization?

Reply:
It's due to the make_fx(tracing_mode="symbolic") call in L282. For eager functionalization and tracing_mode="real", they are both good.

Reply:
What's wrong with tracing_mode="symbolic"? The symbool thing? Or are there more issues?

Reply:
Yeah, the symbol thing.